### ISSUE TITLE
Authentication with Usename and Password for Client.Legacy

### ISSUE THREAD

**Issue by mexxBf** [influxdata/influxdb-client-csharp#64]
I would love to know a way to authenticate with the legacy client. I want to use flux to query data from an InfluxDB 1.7.10. 

In the examples I could not find any clues on how to do it.

Thank you.

**Comment by bednar** [comment]
Hi @mexxBf,

Currently the legacy client doesn't supports [authenticated requests](https://docs.influxdata.com/influxdb/v1.7/administration/authentication_and_authorization/#authenticate-requests), but will fix that as soon as possible... probably in milestone [1.6.0](https://github.com/influxdata/influxdb-client-csharp/milestone/9).

Regards

**Comment by mexxBf** [comment]
Hi @bednar,

is there any ETA to version 1.6.0? Its a showstopper for me, so I'am curious :)

Thanks

**Comment by bednar** [comment]
Hi @mexxBf, 

version 1.6.0 will be released probably this Friday. 

Regards

**Comment by bednar** [comment]
Hi @mexxBf,

The issue is fixed in [1.6.0 milestone](https://github.com/influxdata/influxdb-client-csharp/milestone/9). 

If you would like to use a preview version use our dev build:

```
dotnet add package InfluxDB.Client.Flux --version 1.6.0-dev.443
```

Regards

**Comment by Wawszczak** [comment]
Unfortunatelly this method of authentication is not secure, because it adds credentials to URL not to request headers.
